app/views/users/show.html.erb

```diff
@@ -6,10 +6,11 @@
 </div>
 
```

Comment:
The app has an error on the start which stops it from running. Kindly see the error below and make sure the app is running.
![users](https://user-images.githubusercontent.com/35504149/152549391-904d4c7a-f2fb-4717-9d3b-b65f2e570608.png)